src/features/TaskScreen/TaskScreen.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { View, Dimensions, StyleSheet} from 'react-native';
+import { TabView, SceneMap } from 'react-native-tab-view';
+
+const ToDoRoute = () => (
+    // put component here, or direct feed with tasks
+    <View style={[styles.scene, { backgroundColor: '#ff4081' }]} />
+);
+
+const InProgressRoute = () => (
+    // put component here, or direct feed with tasks
+    <View style={[styles.scene, { backgroundColor: '#673ab7' }]} />
+);
+
+const VerifyRoute = () => (
+    // put component here, or direct feed with tasks
+    <View style={[styles.scene, { backgroundColor: '#ff4081' }]} />
+);
+
+const DoneRoute = () => (
+    // put component here, or direct feed with tasks
+    <View style={[styles.scene, { backgroundColor: '#673ab7' }]} />
+);
+
+const initialLayout = { width: Dimensions.get('window').width };
+
+function TaskScreen() {
+    const [index, setIndex] = React.useState(0);
```

Comment:
If you import react like that: `import React, { useState } from 'react'` ... you would not need `React.` here.